keps/sig-node/2535-ensure-secret-pulled-images/kep.yaml

```diff
@@ -18,11 +18,11 @@ approvers:
 prr-approvers:
```

Comment:
This needs to move to the dedicated PRR file.

Reply:
ok is already in .. https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-node/2535.yaml#L3 and will remove from kep.yaml